frontend/integration-tests/views/sidenav.view.ts

```diff
@@ -1,10 +1,14 @@
 import { $$, by, browser, element, ExpectedConditions as until } from 'protractor';
 
-export const navSectionFor = (name: string) => element(by.cssContainingText('.pf-c-nav > .pf-c-nav__list > .pf-c-nav__item', name));
+export const navSectionFor = (name: string) => element(by.linkText(name));
```

Comment:
This isn't specific enough since there might be other links on the page with the same text.

It would be better to add a `data-test-` attribute if we can.

Reply:
Do you mean to leave it as it was before:
 element(by.cssContainingText('.pf-c-nav > .pf-c-nav__list > .pf-c-nav__item', name));

it failed to navigate this way

Reply:
No, I mean add a `data-test-<something>` attribute into the markup and use that in the tests

Reply:
I guess adding data-tests attributes to markup is something done by developers, right ?

Reply:
I read about it and got your point. these data* need to be added to markup by developers, so we ( QE automation) be able to use them.

I need to raise this topic on DFG and see how responsive they would be to such changes/additions.

The question is how can we utilize current selectors to solve this issue meanwhile ?

What made this not working now:
element(by.cssContainingText('.pf-c-nav > .pf-c-nav__list > .pf-c-nav__item', name));

Reply:
What if I limit searching locators from pf-c-nav__list, there is only like that on page:
element(by.css('.pf-c-nav__list')).element(by.linkText(name))

I tested it and it is working